x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/parser/ExpressionBuilder.java

```diff
@@ -717,7 +720,11 @@ public Literal visitParamLiteral(ParamLiteralContext ctx) {
         }
         // otherwise we need to make sure that xcontent-serialized value is converted to the correct type
         try {
-            return new Literal(source, converterFor(sourceType, dataType).convert(param.value), dataType);
+            if (SqlDataTypeConverter.canConvert(sourceType, dataType) == false) {
```

Comment:
Sure will do. 
But for the future, just to understand the reason: is it because no other `canConvert` (like `ql`'s) should be used in the file and a static import would make that more obvious? (Asking since the function is used currently just once in the entire file.)